frontend/app/src/App.tsx

```diff
@@ -840,6 +841,8 @@ export class App extends PureComponent<Props, State> {
 
   handlePageProfileMsg = (pageProfile: PageProfile): void => {
     const pageProfileObj = PageProfile.toObject(pageProfile)
+
+    const browserInfo = getBrowserInfo()
     this.metricsMgr.enqueue("pageProfile", {
```

Comment:
does segment need to be updated to know the expected shape of this object, or does it just take any JSON?

Reply:
I think Segment takes any JSON. However, our data pipelines may need to be adapted to handle the new data. Maybe you should reach out to the data team to double-check if this could cause any trouble.